apps/src/applab/reducers.js

```diff
@@ -100,6 +101,12 @@ function instructions(state, action) {
     });
   }
 
+  if (action.type === ActionType.SET_INSTRUCTIONS_MAX_HEIGHT) {
+    return _.assign({}, state, {
+      maxHeight: action.maxHeight
```

Comment:
In this case (and some of the cases above) you may want to take care to return the original state object if the value of the given property hasn't changed.